gwlfe/datamodel.py

```diff
@@ -301,6 +301,19 @@ def defaults(self):
             'n13bdp': 0,
             'n13cdp': 0,
             'RetentEff': 0,
+
+            # Referenced in WriteOutputFiles
+            'SumSedDelivRatio': 0,
+            'SumTotArea': 1,
+            'SumPrecipitation': np.zeros(12),
+            'SumEvapoTrans': np.zeros(12),
+            'SumGroundWater': np.zeros(12),
+            'SumRunoff': np.zeros(12),
+            'SumStreamFlow': np.zeros(12),
+            'SumPtSrcFlow': np.zeros(12),
+            'SumTileDrain': np.zeros(12),
+            'SumWithdrawal': np.zeros(12),
+            'AreaSum': np.zeros(12),
```

Comment:
These declarations just need to be moved to the parser class and everything should more or less work with my recent changes in #30.